ivadomed/main.py

```diff
@@ -405,18 +408,33 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
         save_config_file(context, path_output)
 
         # RUN TRAINING
-        best_training_dice, best_training_loss, best_validation_dice, best_validation_loss = imed_training.train(
-            model_params=model_params,
-            dataset_train=ds_train,
-            dataset_val=ds_valid,
-            training_params=context["training_parameters"],
-            path_output=path_output,
-            device=device,
-            cuda_available=cuda_available,
-            metric_fns=metric_fns,
-            n_gif=n_gif,
-            resume_training=resume_training,
-            debugging=context["debugging"])
+        n_gpus = device_count()
+        if n_gpus > 1:
+            logger.info(f"Using {n_gpus} GPUs")
+
+            os.environ['MASTER_ADDR'] = context["master_addr"] if "master_addr" in context else "localhost"
```

Comment:
This line can be re-written as 
`os.environ['MASTER_ADDR']  = context.get("master_addr", "localhost") `